lib/fastlane/plugin/xcodegen/actions/xcodegen_action.rb

```diff
@@ -42,6 +42,10 @@ def self.details
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :executable,
```

Comment:
Can you add the new option to the README file

Reply:
Sure!